iohk-monitoring/src/Cardano/BM/ElidingTracer.lhs

```diff
@@ -87,13 +88,21 @@ Internal state transitions.
     traceWith (toLogObject' tverb tr) ev
     return (Nothing, 0)
   stopeliding tverb tr ev (Just ev0, count) = do
-    when (count > 1) $ do  -- report the number of elided messages
-      meta <- mkLOMeta (getSeverityAnnotation ev0) (getPrivacyAnnotation ev0)
-      traceNamedObject tr (meta, LogValue "before next, messages elided" (PureI $ toInteger (count - 1)))
+    when (count > 1) $  -- report the number of elided messages
+      reportelided tverb tr ev0 count
```

Comment:
Why in this module lowcase-names are used? I mean `stopeliding` instead of `stopEliding`, `reportelided` instead of `reportElided`, `defaultelidedreporting` instead of `defaultElidedReporting`? It's a little bit harder to read, do you agree?